worker/internal/benthos/transformers/generate_street_address.go

```diff
@@ -20,16 +20,28 @@ type Address struct {
 }
 
 func init() {
-	spec := bloblang.NewPluginSpec().Param(bloblang.NewInt64Param("max_length"))
+	spec := bloblang.NewPluginSpec().Param(bloblang.NewInt64Param("max_length")).Param(bloblang.NewInt64Param("seed"))
```

Comment:
This needs to be marked as optional so that it can be omitted from the bloblang interface.
```suggestion
	spec := bloblang.NewPluginSpec().Param(bloblang.NewInt64Param("max_length")).Param(bloblang.NewInt64Param("seed").Optional())
```

Reply:
updated